pandas/util/_decorators.py

```diff
@@ -332,7 +333,7 @@ def make_signature(func):
         n_wo_defaults = len(spec.args) - len(spec.defaults)
         defaults = ('',) * n_wo_defaults + tuple(spec.defaults)
     args = []
-    for i, (var, default) in enumerate(zip(spec.args, defaults)):
+    for var, default in zip(spec.args, defaults):
```

Comment:
Hmm what are you changing this for?

Reply:
There is no need to use enumerate.

Reply:
Hmm OK thanks. Surprised that isn't picked up by the linter